README.md

```diff
@@ -125,4 +206,10 @@ Assuming variables _n_ and _d_ are defined in advance (but you don't know their
 
 a) Solve it first by making use of a function called _Math.floor_.
 
+Math.floor (9/4) = 2
+
+
 b) Now solve it without calling any functions, using merely operators.  (Hint: you'll need at least the modulo operator _%_.)
+
+
+
```

Comment:
It looks like you have a decent understanding of what you were supposed to do, but just needed a few pointers. I'd be more than happy to explain problems 4 and 5 in office hours sometime if that's something that you're interested in! This stuff is very confusing at first, but coding with other people, and asking as many questions as possible, will definitely help you understand things quicker!